libraries/cbor/src/reader.rs

```diff
@@ -38,9 +38,8 @@ pub enum DecoderError {
 }
 
 /// Deserialize CBOR binary data to produce a single [`Value`], expecting that there is no additional data.
-/// Supports arbitrarily nested CBOR (so the [`DecoderError::TooMuchNesting`] error is never emitted).
 pub fn read(encoded_cbor: &[u8]) -> Result<Value, DecoderError> {
```

Comment:
If we keep this function, let's add a comment that it uses maximum nesting of 127.

Reply:
Done.